src/core/src/core_logic/PatchAssessor.py

```diff
@@ -111,10 +111,11 @@ def start_assessment(self):
         return True
 
     def write_assessment_perf_logs(self, retry_count, task_status, error_msg):
-        assessment_perf_log = {Constants.PerfLogTrackerParams.TASK: Constants.ASSESSMENT, Constants.PerfLogTrackerParams.TASK_STATUS: str(task_status),
-                               Constants.PerfLogTrackerParams.ERROR_MSG: error_msg, Constants.PerfLogTrackerParams.PACKAGE_MANAGER: self.package_manager_name,
-                               Constants.PerfLogTrackerParams.RETRY_COUNT: str(retry_count)}
-        self.stopwatch.stop_and_write_telemetry(str(assessment_perf_log))
+        assessment_perf_log = "[{0}={1}][{2}={3}][{4}={5}][{6}={7}][{8}={9}][{10}={11}]".format(
```

Comment:
do these changes required in ConfigurePatchingProcessor as well? In case, changes are needed, pls test in Arc machine as well.

Reply:
Good question. This mostly needed for PatchInstaller. PatchAssessor is mostly deterministic in time taken. CP is similar. It's fine as it's scoped for now, and this PR is mostly a needed improvement prior to the next deployment so want to avoid scope creep that'd delay it.